src/components/CamundaDistributions/DockerCompose.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import DocsVersion from "../DocsVersion";
+
+const DockerComposeBaseURL =
+  "https://github.com/camunda/camunda-self-managed/releases/download";
+
+const DockerCompose = () => {
+  const docsVersion = DocsVersion();
+  const version = docsVersion == "Next" ? "alpha" : docsVersion;
+  return (
+    <a
+      style={{ color: "var(--ifm-link-color)" }}
+      class="theme-doc-version-badge badge badge--secondary"
```

Comment:
I don't think the style/class is necessary; returning the link should format it appropriately.

Reply:
👍 removed it!